InWorldz/InWorldz.Phlox.Engine/LSLSystemAPI.cs

```diff
@@ -7414,7 +7414,7 @@ private Object AutoCastString(string str)
 
         private string iwParseString2ListSub(string str, int trimString, int doCapitalize)
         {
-            if (str == "") return "";
+            if (String.IsNullOrEmpty(str)) return str;
```

Comment:
Similar to above: best case is an avoidance of a null pointer exception and worst case is no real change to functionality.